AnkiDroid/src/test/java/com/ichi2/anki/RobolectricTest.kt

```
@@ -105,7 +105,11 @@ open class RobolectricTest : CollectionGetter {
             RustBackendLoader.loadRsdroid(backendPath)
         } else {
             // default (no env variable): Extract the backend testing lib from the jar
-            RustBackendLoader.init()
+            try {
+                RustBackendLoader.init()
+            } catch (e: UnsatisfiedLinkError) {
+                Timber.e("Please update the environment variables with following commands : export ANKIDROID_BACKEND_PATH=\"{path-to-directory}/Anki-Android-Backend/rslib-bridge/target/aarch64-apple-darwin/release/librsdroid-arm64.dylib\" export ANKIDROID_BACKEND_VERSION=\"0.1.10\"")
```

* We want to throw an exception, Timber ignores the error
* We only want to do this if `e.message` mentions `arm64e`
* We don't want to hardcode the backend version, as that will change
  * The data should come from the tag 
* We want the directory to the `dylib`, not including `/Anki-Android-Backend/rslib-bridge/target/aarch64-apple-darwin/release/`